hack/csv-generator.go

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
I would vote not to do that, because `operator-sdk` might expect `main.go` in the root directory if we try to generate webhooks (or anything else that adds code) in the future

Reply:
Ok, good point.